base_agent/dialogue_objects/location_helpers.py

```diff
@@ -97,4 +97,4 @@ def interpret_point_target(interpreter, speaker, d):
         interpreter, speaker, mems, steps, reldir
     )
     #    loc, _ = compute_locations(interpreter, speaker, mems, steps, reldir)
-    return (loc[0], loc[1] + 1, loc[2], loc[0], loc[1] + 1, loc[2])
+    return (loc[0], loc[1], loc[2], loc[0], loc[1]+1, loc[2])
```

Comment:
am worried if there isn't a +1 on the first loc[1] its going to do weird things to mobs/players when it points (which in mc is flashing a block into and out of existence).